Nits and wording improvements. (#131)
tlswg/dtls13-spec: draft-ietf-tls-dtls13.md

  - server: The endpoint which did not initiate the DTLS connection.

  - CID: Connection ID
  
  - MSL: Maximum Segment Lifetime

The reader is assumed to be familiar with the TLS 1.3 specification since this
document is defined as a delta from TLS 1.3. As in TLS 1.3 the HelloRetryRequest has
the same format as a ServerHello message but for convenience we use the term
HelloRetryRequest throughout this document as if it were a distinct message.

> The reader is also used to be familiar with {{I-D.ietf-tls-dtls-connection-id}}
> since this document applies the CID functionality to DTLS 1.3. 

Reviewer: ```suggestion
The reader is also as to be familiar with {{I-D.ietf-tls-dtls-connection-id}}
as this document applies the CID functionality to DTLS 1.3. 
```

A recipient of a DTLS message needs to select the correct keying material
in order to process an incoming message. With the possibility of message

>  loss and re-order, an identifier is needed to determine which cipher state

Reviewer: ```suggestion
 loss and re-ordering, an identifier is needed to determine which cipher state
```
